docs/dev/starting-a-mod.md

```diff
@@ -24,37 +24,63 @@ It does come with some minor drawbacks such as it using "LethalCompanyTemplate"
 
 It is important that you remove all unused template code from the project before distributing your mod.
 
-### Creating your project {#create-project}
+### Using the dotnet template {#using-dotnet-template}
 
-First things first, you'll need to create your project. If you've not done so already, we recommend running the following command in a console to add some BepInEx templates for new projects:
+First things first, you'll need to create your project. If you've not done so already, we recommend running the following command in a console to add some Lethal Company templates for new projects:
 
-```cmd
-dotnet new -i BepInEx.Templates --nuget-source https://nuget.bepinex.dev/v3/index.json
+```shell
+dotnet new -i Xilophor.LCModTemplates
 ```
 
 Next, you'll want to create a new project (sometimes called "solution", in CSharp). There are two main ways to do this.
 
-#### Using an IDE (more control) {#using-ide}
+#### Using an IDE (easier) {#using-ide}
 
-Depending on your IDE, this process will look slightly different. You'll want to give the solution the name of your soon-to-be mod. If given the option to use a template (you may want to google for *"how to use template in Visual Studio"* or *"how to use template in Rider"*), use the `BepInEx 5 Plugin Template`.
+Depending on your IDE, this process will look slightly different. You'll want to give the solution the name of your soon-to-be mod. If given the option to use a template (you may want to google for *"how to use template in Visual Studio"* or *"how to use template in Rider"*), use the `Lethal Company Harmony Mod Template`.
 
-#### Using the console (simpler) {#using-console}
+::: warning
+Note that Visual Studio has issues with both the `NuGetPackages` setting and the `UseNetcodePatcher` setting. Anything set in `NuGetPackages` will be ignored by Visual Studio; and if you set `UseNetcodePatcher` to true, you will have to manually run the `install-netcode-patcher.cmd` file.
```

Comment:
I guess this is more relevant to the quirks of Visual Studio (with my template specifically - though these quirks exist with any template using the same provided methods). It can be removed or moved to a details container.